packages/contracts-por/package.json

```diff
@@ -5,13 +5,20 @@
   "private": true,
   "scripts": {
     "clean": "rm -rf ./build && hardhat clean",
+    "lint": "yarn lint:sol && yarn lint:ts",
+    "typecheck": "tsc --noEmit",
+    "lint:sol": "solhint 'contracts/**/*.sol' && prettylint 'contracts/**/*.sol'",
+    "lint:js": "eslint '{test,scripts}/**/*.js'",
+    "lint:ts": "eslint '{test,scripts}/**/*.ts' -c .eslintrc.typescript.js",
+    "lint:fix": "prettier 'contracts/**/*.sol' --write --loglevel error && yarn lint:ts --fix",
     "prebuild": "yarn clean",
     "build:hardhat": "hardhat compile",
     "build:typechain": "typechain --target ethers-v5 --out-dir build/types 'build/*.json'",
     "build": "yarn build:hardhat && yarn build:typechain && mars",
     "preflatten": "rm -rf custom_flatten",
     "flatten": "waffle flatten .waffle.json",
-    "test": "mocha 'test/**/*.test.ts'"
+    "test": "mocha 'test/**/*.test.ts'",
+    "checks": "yarn lint && yarn test && yarn test:integration"
```

Comment:
There's no `test:integration` script in this `package.json file`. Looks like a copy-paste error.

Reply:
Removed now.